docs/release-notes.md

```diff
@@ -2,6 +2,17 @@
 
 This page summarizes changes for each major update in Proton and Timeplus Cloud, including new features and important bug fixes.
 
+## Jan 8, 2023
+
+*Proton:*
+  * We've added a new example in the [proton/examples](https://github.com/timeplus-io/proton/tree/develop/examples) folder for [Coinbase](https://github.com/timeplus-io/proton/tree/develop/examples/coinbase).
+  * (v1.3.30) Kafka external streams now support SCRAM-SHA-256/512.
```

Comment:
Need to be more specific: 
When you create the external stream, set sasl_mechanism to SCRAM-SHA-512, SCRAM-SHA-256, or PLAIN (this is  the default value if you don't set sasl_mechanism). Learn more with examples. https://docs.timeplus.com/proton-kafka#create-external-stream


sasl_mechanism

Reply:
I need to add docs about `Kafka external stream virtual columns` in 1.3.30

Reply:
I need to add docs about `enable base64 function`

Reply:
I need to add docs about ML functions stochastic_linear_regression_state and stochastic_logistic_regression_state `(lower_name == "stochastic_linear_regression_state" || lower_name == "stochastic_logistic_regression_state")`